src/main/io/osd.h

```diff
@@ -233,3 +243,4 @@ void osdSuppressStats(bool flag);
 void setOsdProfile(uint8_t value);
 uint8_t getCurrentOsdProfileIndex(void);
 void changeOsdProfileIndex(uint8_t profileIndex);
+bool osdElementVisible(uint16_t value);
```

Comment:
I don't think this is used anywhere externally (or meant to be), so this shouldn't be here.